infra-as-code/bicep/modules/hubNetworking/hubNetworking.bicep

```
@@ -526,6 +529,7 @@ resource resBastion 'Microsoft.Network/bastionHosts@2023-02-01' = if (parAzBasti
   }
   properties: {
     dnsName: uniqueString(resourceGroup().id)
+    enableTunneling: parBastionTunneling
```

```suggestion
    enableTunneling: (parAzBastionSku == 'Standard' && parAzBastionTunneling) ? parAzBastionTunneling : false
```

adding some logic for ensuring standard has been selected 👍

Can you please test both deploy with standard and basic to ensure all works